Add OpInfo for `torch.nn.functional.gaussian_nll_loss`

ejguan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #67465 Add OpInfo for `nn.functional.cosine_embedding_loss`
* #67464 Add OpInfor for `nn.functional.ctc_loss`
* #67371 Add OpInfo for `nn.functional.poisson_nll_loss`
* #67376 Add OpInfo for `nn.functional.huber_loss`
* #67381 Add OpInfo for `nn.functional.hinge_embedding_loss`
* **#67356 Add OpInfo for `torch.nn.functional.gaussian_nll_loss`**

Differential Revision: [D31970077](https://our.internmc.facebook.com/intern/diff/D31970077)

ejguan committed:
Add OpInfo to `torch.nn.functional.gaussian_nll_loss`

[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `torch.nn.functional.gaussian_nll_loss`"




[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `torch.nn.functional.gaussian_nll_loss`"


Differential Revision: [D31970077](https://our.internmc.facebook.com/intern/diff/D31970077)

[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `torch.nn.functional.gaussian_nll_loss`"


Differential Revision: [D31970077](https://our.internmc.facebook.com/intern/diff/D31970077)

[ghstack-poisoned]

mruberry reviewed (approved):
Cool!

The sample inputs might want to add some eps kwarg testing. Such a thorough enumeration for sample inputs is cool but just having 5-9 hand-written sample inputs is cool, too.

ejguan committed:
Update on "Add OpInfo for `torch.nn.functional.gaussian_nll_loss`"


Differential Revision: [D31970077](https://our.internmc.facebook.com/intern/diff/D31970077)

[ghstack-poisoned]

ejguan committed:
Update on "Add OpInfo for `torch.nn.functional.gaussian_nll_loss`"


Differential Revision: [D31970077](https://our.internmc.facebook.com/intern/diff/D31970077)

[ghstack-poisoned]
